elfpy/math/fixed_point.py

```diff
@@ -241,11 +245,15 @@ def __neg__(self) -> FixedPoint:
         return FixedPoint("-1.0") * self
 
     def __abs__(self) -> FixedPoint:
-        """Enables 'abs()' function"""
+        r"""Enables 'abs()' function"""
```

Comment:
I might be wrong here, but I think it will make a difference for the backticks to render as an inline code block. If the docstring has only text you don't need `r`, but if it has any sort of sphinx or markdown or whatever then we should use `r`. And since it doesn't really hurt to use it I think it's fine to just use it everywhere.